lighthouse-core/runner.js

```diff
@@ -36,6 +37,13 @@ class Runner {
       return Promise.reject(err);
     }
 
+    const sentryContext = Sentry.init(opts);
```

Comment:
Not clear what you're proposing to me since we need the options before we can start sending the errors. If you're talking about buffering the events and sending them after Sentry has been initialized that sounds like future work than initial introduction of error reporting to me :)